apis/sgv2-quarkus-common/src/main/resources/application.yaml

```diff
@@ -0,0 +1,4 @@
+quarkus:
+  log:
+    console:
+      format: "%-5p [%t] %d{yyyy-MM-dd HH:mm:ss,SSS} %F:%L - %s%e%n"
```

Comment:
I don't think this will work? Ohh wow this does work..

Reply:
Yes, tested all 3 apis (docs, rest, and graphql), the log format change is reflected.